src/index.js

```diff
@@ -45,9 +45,9 @@ const Root = () => {
           colorblack="color--black"
           logoname="logo.png"
         />
-        <div className="slider-wrapper">
+        {/* <div className="slider-wrapper ">
           <SliderOne />
-        </div>
+        </div> */}
```

Comment:
It's okay to delete code, since we have the git history.

Can you please remove this since we're no longer using it in the `index.jsx`?